tools/peptideshaker/fasta_cli.xml

```diff
@@ -0,0 +1,63 @@
+<tool id="fasta_cli" name="FastaCLI" version="1.0">
+    <description>
+        Appends decoy sequences to FASTA files
+    </description>
+    <macros>
+        <import>macros_basic.xml</import>
+    </macros>
+    <requirements>
+	<requirement type="package" version="@SEARCHGUI_VERSION@">searchgui</requirement>
+	<requirement type="package" version="3.0">zip</requirement>
+    </requirements>
+    <expand macro="stdio" />
+    <command>
+<![CDATA[
+
+        ## #set $temp_stderr = "searchgui_stderr"
+        #set $bin_dir = "bin"
+
+        mkdir output;
```

Comment:
can you use `&&` instead of `;` this will fail the tool as soon as one command fails and not silently ignore the error as `;` is doing.